{{cookiecutter.project_name}}/dev-docker-compose.yml.default

```diff
@@ -0,0 +1,42 @@
+# Configuration for local development
+#
+# See README for setup and usage instructions
+#
+version: '3'
+
+services:
+  postgres:
+    image: postgres
+    ports:
+      - "5433:5433"
+      - "5432:5432"
+    volumes:
+      - "../db:/db"
+      - "../backup:/backup"
+      - ".:/project"
+    environment:
+      POSTGRES_PASSWORD: {{cookiecutter.project_name}}
+      POSTGRES_USER: {{cookiecutter.project_name}}
+      PGDATA: /db
+
+  redis:
+    image: redis
```

Comment:
same as pg for pinning to a specific version

Reply:
Same fix as above.